### ISSUE TITLE
Update Authorize.net Endpoints

### ISSUE THREAD

**Issue by JustinSainton** [wp-e-commerce/Gold-Cart#8]
See http://www.authorize.net/support/akamaifaqs/?utm_campaign=Akamai%20Initial%20Email%20to%20Developers%20and%20Partners.ht&utm_medium=email&utm_source=Eloqua#newurls

**Comment by mrkaln** [comment]
Is there any news when a new release of Gold Cart will include these changes?

**Comment by JustinSainton** [comment]
No news yet.  We'd gladly take a PR on it, and we intend on having it in our next release, but we don't have a hard date on that yet.

Part of what I'd like to do is to update the entire Authorize.net SDK, remove any hard references to the sandbox/production endpoints in the gateway classes, and refer to the SDK endpoints directly.  That takes a bit more time and testing than simply changing the current hard references.

Hope that helps!

**Comment by mrkaln** [comment]
I assumed the gateway would stop working if these URLS were not updated soon. Will it continue to work for now?

**Comment by JustinSainton** [comment]
Yeah, so that's not quite the case.  Here's what the link in the original post says:

> # Are the current URLs changing or will they be inactive in June 2016?
> 
> No. The existing transaction URLs on secure.authorize.net and api.authorize.net are not changing and will remain in service. They will be directed to Akamai in June 2016, however, they will continue to work as usual unless you have a firewall that has whitelisted our IP address for outbound connections (see "I use a firewall that has whitelisted Authorize.Net's IP address for outbound connections. What do I need to do to use Akamai?" for more information).
